docs/settings.rst

```diff
@@ -1731,6 +1731,25 @@ of reflection attacks. Maximum value is 65535, but values above
   1232 is the largest number of payload bytes that can fit in the smallest IPv6 packet.
   IPv6 has a minimum MTU of 1280 bytes (:rfc:`RFC 8200, section 5 <8200#section-5>`), minus 40 bytes for the IPv6 header, minus 8 bytes for the UDP header gives 1232, the maximum payload size for the DNS response.
 
+.. _setting-upgrade-unknown-types:
+
+``upgrade-unknown-types``
+-------------------------
+
+-  Boolean
+-  Default: no
+
+.. versionadded:: 4.4.0
+
+Transparently upgrade records stored as `TYPE#xxx` and RFC 3597 (hex format)
+contents, if the type is natively supported.
+When this is disabled, records stored in this format cannot be served.
+
+Recommendation: keep disabled for better performance.
+Enable for testing PowerDNS upgrades, without changing stored records.
+
+Not all backends can use this option.
```

Comment:
```suggestion
Not all backends support this option.
```

Reply:
If you do, please add a list of supported backends as well.

Reply:
ah, yes please

Reply:
added!